doc/dev/developer_guide/essentials.rst

```diff
@@ -13,20 +13,19 @@ following table shows all the leads and their nicks on `GitHub`_:
 
 .. _github: https://github.com/
 
-========= ================ =============
-Scope     Lead             GitHub nick
-========= ================ =============
-Ceph      Sage Weil        liewegas
-RADOS     Neha Ojha        neha-ojha
-RGW       Yehuda Sadeh     yehudasa
-RGW       Matt Benjamin    mattbenjamin
-RBD       Ilya Dryomov     dis 
-CephFS    Venky Shankar    vshankar
-Dashboard Ernesto Puerta   epuertat
-MON       Joao Luis        jecluis
-Build/Ops Ken Dreyer       ktdreyer
-Docs      Zac Dover        zdover23
-========= ================ =============
+========= ================== =============
+Scope     Lead               GitHub nick
+========= ================== =============
+RADOS     Radoslaw Zarzynski rzarzynski 
+RGW       Casey Bodley       cbodley 
+RGW       Matt Benjamin      mattbenjamin
+RBD       Ilya Dryomov       dis 
+CephFS    Venky Shankar      vshankar
+Dashboard Nizamudeen A       nizamia109 
```

Comment:
;)
```suggestion
Dashboard Nizamudeen A       nizamial09 
```

Reply:
What am I missing? I don't see a difference. I've been awake for a very long time, so I might be a little dumb right now.

Reply:
You put `1` instead of the alphabet `L` before `09`

Reply:
It looks so similar...

Reply:
Aw man. Our orthography is almost as dumb as I was in this instance.

Thanks for pointing this out, @nizamial09.

Reply:
no problem!